nursery_store.go

```diff
@@ -181,14 +163,6 @@ var (
 	// the root-level, chain-segmented bucket for each nursery store.
 	utxnChainPrefix = []byte("utxn")
 
-	// lastFinalizedHeightKey is a static key used to locate nursery store's
```

Comment:
With these changes, do we need a migration to clean up old state that will no longer be read?

Reply:
I am not sure. We are on our way to getting rid of the nursery and its touchy state, so it will be cleaned up eventually anyway. On the other hand, priorities change and not migrating the database allows people to downgrade, which will may lead to surprising results. So I think, yes we should migrate.